src/tasks/SingularityHandler.h

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file SingularityHandler.h
+ * @author William Chong (wmchong@stnaford.edu)
+ * @brief This class handles type 1 and type 2 singularity.
+ *          For type 1, the joint task is blended with the force task in the singular space.
+ *          For type 2, an open-loop torque is applied proportional to the dot product between 
+ *          the force task and the singular direction and blended. The torque direction can be either direction
+ *          since the singular direction can move pi/2 either direction to be orthogonal to the force vector
+ *          since the singular direction only needs to be in-line with the force vector.
+ * @version 0.1
+ * @date 2024-01-13
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#pragma once 
```

Comment:
Use header guards
#ifndef ...._H_
#define ...._H_

#endif 

for consistency with the rest of the repo